VoltAir/resources/values-Swahili.properties

```diff
@@ -0,0 +1,17 @@
+finish=Fin
```

Comment:
@diegowald To make the file easier to translate, change its content from Spanish to English.